lib/chef/knife/ec2_ami_list.rb

```diff
@@ -0,0 +1,101 @@
+#
+# Author:: Piyush Awasthi (<piyush.awasthi@msystechnologies.com>)
+# Copyright:: Copyright (c) 2010-2015 Chef Software, Inc.
```

Comment:
if this is a new file from scratch, which it appears to be, the copyright should be just 2017.